includes/abstracts/abstract-wc-order.php

```diff
@@ -1109,7 +1109,7 @@ public function update_taxes() {
 			$item = new WC_Order_Item_Tax();
 			$item->set_rate( $tax_rate_id );
 			$item->set_tax_total( isset( $cart_taxes[ $tax_rate_id ] ) ? $cart_taxes[ $tax_rate_id ] : 0 );
-			$item->set_shipping_tax_total( isset( $shipping_taxes[ $tax_rate_id ] ) ? $shipping_taxes[ $tax_rate_id ] : 0 );
+			$item->set_shipping_tax_total( !empty( $shipping_taxes[ $tax_rate_id ] ) ? $shipping_taxes[ $tax_rate_id ] : 0 );
```

Comment:
Ah, good point! Added pre-commit hooks for `woocommerce` locally as well. Thanks!